skbio/alignment/tests/test_tabular_msa.py

```diff
@@ -493,6 +493,43 @@ def test_reindex_non_hashable_keys(self):
 
         npt.assert_array_equal(msa.keys, keys)
 
+    def test_append(self):
+        msa = TabularMSA([DNA('ACGT'), DNA('TGCA')])
+        msa.append(DNA('AAAA'))
+        self.assertEqual(msa.shape, (3, 4))
+        # TODO: Hack to get last seq. once __getitem__ is implemented use
+        # msa[-1]
+        seq = None
+        for seq in msa:
+            pass
+        self.assertEqual(seq, DNA('AAAA'))
+
+    def test_append_to_empty_msa(self):
+        msa = TabularMSA([])
+        msa.append(DNA('ACGT'))
+        self.assertEqual(len(msa), 1)
```

Comment:
These tests should use `self.assertEqual(msa1, msa2)` to compare `TabularMSA` objects (in order to check shape and other properties).